heal-commands.go

```diff
@@ -321,18 +336,22 @@ type HealingDisk struct {
 	// Copied from cmd/background-newdisks-heal-ops.go
 	// When adding new field, update (*healingTracker).toHealingDisk
 
-	ID            string    `json:"id"`
-	PoolIndex     int       `json:"pool_index"`
-	SetIndex      int       `json:"set_index"`
-	DiskIndex     int       `json:"disk_index"`
-	Endpoint      string    `json:"endpoint"`
-	Path          string    `json:"path"`
-	Started       time.Time `json:"started"`
-	LastUpdate    time.Time `json:"last_update"`
-	ObjectsHealed uint64    `json:"objects_healed"`
-	ObjectsFailed uint64    `json:"objects_failed"`
-	BytesDone     uint64    `json:"bytes_done"`
```

Comment:
Is this a breaking change? you are removing two fields? and its renamed as `Items` ? why ?

Reply:
> Is this a breaking change? you are removing two fields? and its renamed as `Items` ? why ?

emm yes, this is a breaking change. I just wanted to differentiate between objects and items.

Items means objects inside buckets and inside .minio.sys
Objects means only objects inside buckets.

Otherwise the user can be confused by seeing 100 objects healing while he has only 90 objects.

Reply:
I should keeping the old values and mark them as deprecated

Reply:
> I should keeping the old values and mark them as deprecated

I kept those:
```
ObjectsHealed uint64 `json:"objects_healed"` // Deprecated July 2021
ObjectsFailed uint64 `json:"objects_failed"` // Deprecated July 2021
```

Though I don't think that anyone is using it right now. This only breaks people consuming health info in json format

Reply:
Yes lets keep it this way as subnet health analyzer might break.